server/src/main/java/org/elasticsearch/common/util/CuckooFilter.java

```diff
@@ -507,4 +526,205 @@ public boolean equals(Object other) {
             && Objects.equals(this.count, that.count)
             && Objects.equals(this.evictedFingerprint, that.evictedFingerprint);
     }
+
+    //  Forked from Lucene's Packed64 class.vThe main difference is that this version
+    //  can be read from / write to Elasticsearch streams.
+    private static class PackedArray {
+        private static final int BLOCK_SIZE = 64; // 32 = int, 64 = long
```

Comment:
Are all of these lucene code? I was going to say stuff like `Long.SIZE` is more clear here, but if it's lucene code it isn't worth changing it.

Reply:
Yes, those comments are coming from Lucene implementation